app/models/invitation.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Invitation < ApplicationRecord
+  has_secure_token :token
```

Comment:
I like this but I'm more skeptical about it,
- We're limited to tokens of `24` characters in length,
- This does guarantees no collisions to occur (along with the db unique index)  but no counter measures to follow (To be specific enough this is unlikely to happen but the probability of collisions increases with the reserved values size):
If that's normal why don't we consider that pattern for other usage of tokens that requires the user to initiate the generation process (like with reset password and verify account) and save ourselves the overhead of making those `generate_*` methods and catch the collision cases and retry the generation process?

Reply:
Let's talk about this in our meeting today